libSmeagol/timerPocket.py

```diff
@@ -81,7 +81,7 @@ def __callExecuteAction(self) -> None:
     def __startThread(self) -> None:
         self.__running = True
         thread_name = self._getRegistryId()
-        log.info("Setting up thread for " + thread_name)
+        log.info("Setting up thread '%s'", thread_name)
```

Comment:
```suggestion
        log.info(f"Setting up thread: {thread_name}")
```
What info do we gain from this?

Reply:
To make it explicit that a thread has started/wil start.
Should a timer event somehow get botched, this at least show the processing thread was started